man/docker-build.1.md

```diff
@@ -78,10 +78,10 @@ set as the **URL**, the repository is cloned locally and then sent as the contex
    layers in tact, and one for the squashed version.
 
 **--add-host** []
-   Add a custom host-to-IP mapping (host:ip)
+   Add a custom host-to-IP mapping (host:ip, or host=ip)
```

Comment:
Micro-nit; perhaps swap the order to put the preference on `=`;

```markdown
   Add a custom host-to-IP mapping (host=ip, or host:ip)
```

Reply:
Yes, will do. (Also makes it consistent with other text that I'd already fixed.)